reference/content-types/single_category_selection.rst

```diff
@@ -0,0 +1,33 @@
+Single Category selection
+=========================
+
+Description
+-----------
+
+Let you assign one category. Categories can be managed in the settings section of Sulu.
+The selection will be saved as a single id.
+
+Parameters
+----------
+
+.. list-table::
+    :header-rows: 1
+
+    * - Parameter
+      - Type
+      - Description
+    * - request_parameters
+      - collection
+      - Collection of parameters that are appended to the requests sent by the selection.
+
+
+Example
+-------
+
+.. code-block:: xml
+
+    <property name="category" type="single_category_selection">
+        <meta>
+            <title lang="en">Single Category Selection</title>
+        </meta>
+    </property>
```

Comment:
Can we provide also an `Extended Example` how we did it in https://docs.sulu.io/en/2.0/reference/content-types/url.html#extended-example